src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,11 +5,44 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
+    private String color;
+    private String name;
```

Comment:
Don't forget about access modifiers for your custom fields in `Truck` class. Consider making them protected or private to follow encapsulation principle.